src/components/Checkbox.js

```diff
@@ -82,6 +82,13 @@ class Checkbox extends React.Component {
             return;
         }
 
+        const wasChecked = this.props.isChecked;
```

Comment:
Why is this necessary?

Reply:
Old thread https://github.com/Expensify/App/pull/13181#discussion_r1037967395. Oops, I missed https://github.com/Expensify/App/pull/13181#discussion_r1036124301.

Reply:
Why cant we use `this.props.isChecked`? Why make a variable?

Reply:
It was not clear that we want to blur the input when the checkbox is about to be unchecked. 

This onPress call will uncheck the checkbox and the `isChecked` prop will be set to false after `this.props.onPress` is called. 

So others might think that we are blurring the checkbox when it is checked by the user. Thus to make is clear that we are looking at the old state of checkbox, I requested to create a variable `wasChecked`.

Reply:
@hungvu193 Can you please add explanation to the comment?

Reply:
@parasharrajat Sure.
@Luke9389 Do you agree?

Reply:
But I also have the comment below:
```javascript
        // Make sure the checkbox gets unfocused when it's about to be unselected
        if (this.state.isFocused && wasChecked) {
            this.onBlur();
        }

```
 Do you think that's necessary to add another comment about `wasChecked` variable?

Reply:
This comment is not enough to explain why we are doing `Make sure the checkbox gets unfocused when it's about to be unselected`. So you should modifying this comment to add that. 

> add another comment about wasChecked variable?

I am not asking to add a comment behind `wasChecked` but the above.

Reply:
@parasharrajat Thank you, I updated the comment.

Reply:
Oh ok got it. Thanks!